e2e/README.md

```diff
@@ -0,0 +1,3 @@
+# End to End testing
+
+Please find documentation for E2E testing [here](../../docs/Test.md#end-to-end-testing).
```

Comment:
Would suggest having a brief summary here of what this directory contains (along with links to sub folder readmes)